spec/lib/events/s3_importer_spec.rb

```diff
@@ -16,15 +16,16 @@ def gzipped_file(contents)
   let(:file) { gzipped_file(file_contents) }
   let(:file_contents) do
     <<-CSV.strip_heredoc
-      id,action,payload,user_uid,created_at,updated_at,request_id,content_id
-      10,Publish,"{""content_id"":""2cc503f1-5fef-4fac-8381-63f6689cd6a2""}",,2015-11-11 09:55:00 UTC,2015-11-11 09:55:00 UTC,,2cc503f1-5fef-4fac-8381-63f6689cd6a2
-      11,Publish,"{""content_id"":""8bc0c1dd-4842-4283-a123-5671a34b67eb""}",e676cb22-0c0c-4534-9514-a65ebcc7a9e2,2015-11-11 14:00:00 UTC,2015-11-11 14:00:00 UTC,2027-1477408502.282-80.194.77.100-1361,8bc0c1dd-4842-4283-a123-5671a34b67eb
+      id,action,temp_payload,payload,user_uid,created_at,updated_at,request_id,content_id
+      10,Publish,"{""content_id"":""2cc503f1-5fef-4fac-8381-63f6689cd6a2""}","{""content_id"":""2cc503f1-5fef-4fac-8381-63f6689cd6a2""}",,2015-11-11 09:55:00 UTC,2015-11-11 09:55:00 UTC,,2cc503f1-5fef-4fac-8381-63f6689cd6a2
+      11,Publish,"{""content_id"":""8bc0c1dd-4842-4283-a123-5671a34b67eb""}","{""content_id"":""8bc0c1dd-4842-4283-a123-5671a34b67eb""}",e676cb22-0c0c-4534-9514-a65ebcc7a9e2,2015-11-11 14:00:00 UTC,2015-11-11 14:00:00 UTC,2027-1477408502.282-80.194.77.100-1361,8bc0c1dd-4842-4283-a123-5671a34b67eb
```

Comment:
Do we need to add `temp_payload` to the CSV file? Because that will break importing the existing CSV files while we have this field. It seems like because we do [`event.update!`](https://github.com/alphagov/publishing-api/blob/e542e87669a9ab1a71f5f45c4de002b746dbb332/lib/events/s3_importer.rb#L10) the `before_save` should meant that `temp_payload` gets assigned anyway.

Reply:
I _think_ I fixed that now!

Reply:
I think this might have come back in a rebase!

Reply:
Urrgh yeah I'll get rid of it